test/e2e/cypress/plugins/index.js

```diff
@@ -37,13 +46,15 @@ module.exports = (on, config) => {
           })
           .end();
 
-      agents.forEach((agentId) => {
-        heartbeat(agentId);
-        let interval = setInterval(
-          () => heartbeat(agentId),
-          heartbeat_interval
-        );
-        heartbeatsIntervals.push(interval);
+      sleep(500).then(() => {
```

Comment:
**issue:** ‏Do you think we can better understand why it is needed, to remove it?
Sleeps on arbitrary time spans are often cause of flakiness, as they may depend on the running machine

Reply:
I'm not a fan of arbitrary waits either and I know they should be avoided as a good practice, for the moment I don't have a conclusive answer tbh but I noticed that with a bit of delay the hosts_overview test flakiness is reduced a lot so I added it as a contingency measure, we can remove it if you want or even we can try the test flakiness tool designed by @gagandeepb then try with and without the arbitrary sleep and compare results.